app/View/Elements/tutorial_content.ctp

```diff
@@ -16,7 +16,7 @@
     <div id="navigation" class="clearfix">
       <div id="prev-navigation">
 			<?php echo $this->Html->link(
-				__('Previous'),
+				__(''),
```

Comment:
It just occurred to me that if we're going to put a blank string here, we shouldn't run the translation function (`__()`)... nothing to translate. @caosborne89